FppTest/state_machine/test/ut/TesterHelpers.cpp

```diff
@@ -0,0 +1,27 @@
+// ======================================================================
```

Comment:
We should use the auto helpers pattern and remove this file.

Reply:
Fixed